manual_correction.py

```diff
@@ -258,6 +258,12 @@ def get_parser():
              }\n
              """),
     )
+    parser.add_argument(
+        '-task-name', type=str, required=False, default='Manual',
+        help=
+        "R|The name of the task being done, which will be added in the json file. It must be added between \"\"."
+        " For example: \"Manual correction of lesions generated from model XXX\" Default: \"Manual\"g.",
```

Comment:
To track such workflow (automatic segmentation by a model + manual correction), we add two entries into the JSON file (one for the model, the second for the rater). See [here](https://intranet.neuro.polymtl.ca/data/dataset-curation.html#json-sidecars).

```json
{
    "SpatialReference": "orig",
    "GeneratedBy": [
        {
            "Name": "sct_deepseg_sc",
            "Version": "SCT v6.1"
        },
        {
            "Name": "Manual",
            "Author": "Nathan Molinier",
            "Date": "2023-07-14 13:43:10"
        }
    ]
}
```

Reply:
You can specify the model's details using the `-json-metadata` arg, see [L246-L260](https://github.com/spinalcordtoolbox/manual-correction/blob/main/manual_correction.py#L246-L260)